### DISCUSSION TITLE
Delete a customers card?

### DISCUSSION THREAD

**Discussion by osseonews** [swellstores/community#15] [Help] [answered]
As far as I can tell, it is currently not possible to delete a customer's credit card from the Swell Admin Dashboard. However, very often a customer card has expired, and they provide you with a new one. We need a way to then delete the old customer card. Is there any code we can enter in the Console to delete a specific card for a customer or maybe I'm missing something in the UI for the dashboard to delete a card? Thanks.

**Comment by steven-k1** [comment, marked as answer]
@osseonews It is possible to remove a saved card from a customer's account via the dashboard.

1. Navigate to the 'Customers' tab via the admin dashboard.
2. Search for the indicated customer's account you want to update.
3. Once on their account page, on the right-hand side, you will want to select 'Edit' for the card on file. 
4. This will open the edit billing card tab, where you will be presented with the cards on file for that customer. 
5. From there, select the card you wish to edit.
6. At the bottom of the tab for the selected card, you will see three little dots ( ... ).
7. Click on this, and you will be prompted to remove the card. (screenshot below for reference)


I can understand how the option at the bottom can be missed. I will forward this feedback to our team to review and discuss if there is a better way to present this.


![image](https://user-images.githubusercontent.com/77881204/205347945-4ab74153-3a05-4d14-96ba-f5ab2352caf4.png)

**Reply by osseonews** [reply]
Thanks. It literally took me 10 minutes to find this, even after reading your post. Can you please just have them add a "Delete" link/button next to the "Edit" on the credit card page?
